code/src/components/ItemList.js

```diff
@@ -0,0 +1,46 @@
+//ItemList.js stores all tasks/todos
+import React from "react";
+import { useSelector } from "react-redux";
+import { ItemsAll } from "components/ItemsAll";
+
+import styled from "styled-components";
+
+export const ItemList = () => {
+  const allTasks = useSelector((store) => store.tasks.list);
+  const wholeList = useSelector((store) => store.tasks.list.tasks);
+
+  if (wholeList.length === 0) {
```

Comment:
Also love that you added this for the empty list, looks great and the solution is very nice and simple 😍